src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -95,10 +112,15 @@ public void satisfiesEqualsContract() throws IOException {
                 )
             )
         );
-        MatcherAssert.assertThat(
-            new RqMultipart.Base(req),
-            Matchers.equalTo(new RqMultipart.Base(req))
-        );
+        final RqMultipart.Base one = new RqMultipart.Base(req);
+        final RqMultipart.Base two = new RqMultipart.Base(req);
+        try {
+            MatcherAssert.assertThat(one, Matchers.equalTo(two));
+        } finally {
```

Comment:
@raphaelln one more reminder, please update the following comment with username at the beginning:
> I believe the code inside finally needs to be executed, because without the close of the body of all the parts, the files will be on disk permanently.